.circleci/config.yml

```diff
@@ -60,6 +65,59 @@ commands:
           command: |
             sudo apt-get update
             sudo apt-get install -y qemu-user-static binfmt-support
+  sign-images:
+    parameters:
+      sign_arn:
+        type: string
+      sign_session:
+        type: string
+        default: "sign-docker-images"
+      sign_profile_name:
+        type: string
+        default: "default"
+    steps:
+      - run:
+          name: Install Cosign
+          command: |
+            url="https://github.com/sigstore/cosign/releases/latest/download/cosign-linux"
```

Comment:
Should we be pinning the version here instead of using latest? How stable is cosign regarding introducing break changes etc?

Reply:
looks like they do not have many breaking changes unless it's a major version update and it seems like the features we are using are generally untouched